### DISCUSSION TITLE
GitHub Universe 2020 Day 3: What did you learn?

### DISCUSSION THREAD

**Discussion by siddharthdayalwal** [EddieHubCommunity/support#539] [Conference Notes]
From the GitHub Universe 2020 please share what you learnt or any tools you want to learn more about below :point_down: 
![image](https://user-images.githubusercontent.com/41017419/101760057-5cece580-3b00-11eb-99d8-eab32325fc6f.png)

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 3](https://www.twitch.tv/githubeducation)**  🎉

# **`The virtual challenge: grow your community online`** :dart:   
> Get all [GitHub Education Benefits](https://education.github.com/)
- - -

## Students leading the way: Campus Experts
**Juan Pablo Flores - Program Manager, Campus Experts, GitHub**
> Are you interested in building or growing a technical community in your college? The long wait is over, as the Campus Experts Program will soon be accepting applications. The GitHub Campus Experts Program supports students to better serve their local technical communities. Campus Experts help close the gap between industry and academia, create new opportunities for their peers, and position their institution within a global community of student leaders. In this talk, you’ll learn more about the program—from application dates and processes to how to prepare yourself—and be part of the next batch of GitHub Campus Experts.

Gist of the talk :studio_microphone:  


![image](https://user-images.githubusercontent.com/41017419/101820251-51bfa700-3b4c-11eb-9992-5bed0407e35a.png)

> GitHub Campus Experts Program
- Campus Experts are student leaders that strive to build diverse and inclusive spaces to learn skills, share their experiences, and build projects together. They can be found across the globe leading in-person and online conferences, meetups, and hackathons, and maintaining open source projects.
- Learn the skills to build and grow diverse technology communities on campus with training, mentorship, and support from GitHub.
- As local leaders, Campus Experts know the challenges students on their campuses face. With the GitHub Campus Experts training, you’ll learn technical and professional skills—like public speaking, technical writing, community leadership, and software development—that will help you build a strong technical community, teach valuable skills, create new opportunities for your student community, and position your institution within a global community of student leaders.

- 30% growth in 2020 | New countries joined
- **GitHub Camps Experts will be celebrating 5th anniversary in 2021**

> 2020 year was different.....
- New source of opportunities to grab
- No boundaries for students
- [Git Remote Education](https://education.github.com/graduation/)
  - An online graduation ceremony to celebrate graduates of the class of 2020. Join us in walking the stage with your global community. We'll be hearing from special guests, giving out exclusive swag, and highlighting student stories and projects from around the world.
  - 4,000 Pull Requests
  - 2,600 Students
- [MLH HackCon](https://hackcon.mlh.io/)
  - Hackcon is the only conference where hackathon organizers and hacker community leaders can come together, learn from each other, and build relationships. We bring together organizers from Europe, Asia, and North America. This is a unique opportunity to meet community leaders from around the world just like you.
- Content became king
  - Podcasts
  - Blogs
  - Videos, etc.

### Changes with Program Applications and Training
> GitHub Campus Experts Applications
 - Getting to know you better
   - Growth & Potential
   - Motivation
   - Contributions
   - Character
 - How it works?
   - Validate your student status (Get GitHub Student Developer Pack)
   - Fill your application (will open twice a year. 2 weeks after closing you will get status)
   - Record your video resume
     - Submit 3-5 min video into yourself and your community
     - No high production needed, - alternative submission methods
   - Welcome to the Program 🚩
   - Complete the Training

> GitHub Campus Experts Training
- Learning together
 - Train with fellow experts for 6 months
 - Campus Experts mentor-ship
 - Catch-ups and events
 - Meet other trainees

> Application Tips
 - Get GitHub Student Developer Pack
 - Let us know 
    - about yourself
    - your goal
    - your community
    - role you see yourself as
- Apply as soon as possible
- Be 18+ years of age
- Be enrolled in a post-secondary formal education institution.
- Be a GitHub user for at least six months.

> Application Opens
- Applications to the program open twice a year, in February and August. 
- The application remains open for a month 
- You can apply once every application cycle.

> Visit [GitHub Campus Experts](https://education.github.com/students/experts) Program Website 🚩

> Have additional questions?
- You can reach out for more information through the [support form](https://support.github.com/contact/education).
- - -
If you have any questions, reach out to **[@juanpflores_](https://twitter.com/juanpflores_)**

> Do add your notes as well :smile:

**Reply by eddiejaoude** [reply]
These are amazing notes!! 👍 ❤️

**Reply by siddharthdayalwal** [reply]
Thank You :rocket:

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 3](https://www.twitch.tv/githubeducation)**  🎉

# **`The virtual challenge: grow your community online`** :dart:   
> Get all [GitHub Education Benefits](https://education.github.com/)
- - -

## Learning to code: the good, the hard, and the exhilarating
**Sonia John - Lead Blockchain Developer, Porium**
> In this session, we will cover the benefits of learning to code, how to apply the principles of computational thinking in your code journey, and the place of communities for developers.

Gist of the talk :studio_microphone:  

![image](https://user-images.githubusercontent.com/41017419/101813938-63508100-3b43-11eb-91a6-de1b3ca35496.png)

> Why learn to code?
- Software is everywhere
  - Making soft development one of the most bankable skill.
- Secondary skills
  - Develop secondary skills that compliment other problem solving skills.

> Computation Thinking
- Computation Thinking is a way of solving problems where a solution (of which there may be many), can be executed by a computer.
- It's important to remember that "the creativity that one can explore in software development is only limited by creativity of human thought" - Jeanette Wing
- **Principles**
  - Decomposition
    - Break down complexities into smaller more manageable problems.
    - The next time you review your  project's user story, you're in your codebase or creating your release timeline, look out for the types of decomposition you could use.
  - Pattern Recognition
    - Map similarities and differences among decomposed problems to find the commonalities of repetition.
    - The most common use cases of pattern recognition are in the design and development of data analysis solutions. Natural Language Processing, Document Clarification and Facial Recognition.
  - Abstraction
    - Identity the complexities in your solution and split them into multiple interconnects functions.
    - The identification of important information will enable you to leave out of unnecessary code, you'll be able to write documentation for your solution and of course also be able to leave code comments. 
  - Algorithmic Thinking
    - Design algorithms that can be determine the appropriate steps to tale and organizing them into a series of instructions for solving a problem correctly.
    - Being able to apply all these solutions within your code development and developer journey will enable you to share your solutions with users quicker, you'll improve the quality of the softwares you bring to market.

We have millions of software examples built on these principles for instance, the application of Computation Thinking in cryptography

> Why Join a Community?
- Being a member of community is one of the best outcomes of learning to code.
- Sonia John came across some amazing online communities with some of the best people to work with, who have the best attitude when it comes to solving problems and who apply general industriousness to any sort of work.

- - -
If you have any questions, reach out to **[@realChainLife](https://twitter.com/realChainLife)**

> Do add your notes as well :smile:

**Reply by realChainLife** [reply]
Great note, thanks @siddharthdayalwal

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 3](https://www.twitch.tv/githubeducation)**  🎉

# **`The virtual challenge: grow your community online`** :dart:   
> Get all [GitHub Education Benefits](https://education.github.com/)
- - -

## Teachers guiding teachers: Campus Advisors
**Arelia Jones - Developer Program Manager, Teachers, GitHub**
> Arelia Jones is excited to announce the new and improved GitHub Campus Advisors program. This program is made up of teachers who share best practices with their students and colleagues. You’ll learn about opportunities to interact with GitHub and ways that the advisors can support teachers like you. Find out how to join a community of educators who champion the use of real-world tools at their schools.

Gist of the talk :studio_microphone:  

> [GitHub Teachers](https://education.github.com/teachers)
- You train the next generation of software developers.
- That's a big job. Teach your students with the industry-standard tools, so they can shine on technical teams.

![image](https://user-images.githubusercontent.com/41017419/101820109-2a68da00-3b4c-11eb-8c9a-e07c8b5be8a7.png)

> [GitHub Campus Advisors](https://education.github.com/teachers/advisors)
- Launched in early 2018.
- As a GitHub Campus Advisor, you’re not only a master of Git and GitHub, but you champion the use of real-world tools at your school.
- Campus Advisors know that version control is an essential skill for STEM careers and related professions. Campus Advisors help their communities thrive by:

- How it works?
  - Get your toolbox ready
The Teacher Toolbox is an essential tool in completing the Campus Advisor program. On top of that, enjoy access to 30+ premium developer tools for as long as you’re an instructor.
  - Complete the application
Find the application link on the Campus Advisor offer in the Teacher Toolbox. Tell us about yourself and how you use GitHub in your classroom. Let us know how you are sharing best practices and how we can help.
  - Chat with us
After completing your application you’ll have a brief 30 minute interview with someone on the Education team so we can learn more about your school and your students.

- You are a good fit if you are/have 
  - Comfortable using Git and GitHub
  - GitHub account for at least six months.
  - Classroom teacher who teaches students 
  - Using GitHub in the classroom for at least one semester.
  - Verified for GitHub Education for teachers.

> [GitHub Classroom Getting Started Guide](https://www.youtube.com/watch?v=xVVeqIDgCvM&list=PLIRjfNq867bewk3ZGV6Z7a16YDNRCpK3u)

- - -
If you have any questions, reach out to **[@arelia](https://twitter.com/arelia)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 3](https://www.twitch.tv/githubeducation)**  🎉

# **`The virtual challenge: grow your community online`** :dart:   
> Get all [GitHub Education Benefits](https://education.github.com/)
- - -

## Is your campus making you real-world ready?
**Dhiraj Gyani - Director: Education India, GitHub**
> Dhiraj is excited to be talking to students and teachers across the globe and introduce to them how GitHub Education can make them industry ready and prepare them to dive into the real world and make a difference. With developers becoming the most sought-after community in the world, Dhiraj will be talking in detail about the how can students and teachers start early and leverage GitHub stack. These will be coupled with some real stories of evangelization by users across the globe.

Gist of the talk :studio_microphone:  

> [GitHub Campus Program](https://education.github.com/schools)
- Teach with industry-standard tools, across all your departments and students, for free
- With the GitHub Campus Program, your school will get a package of technical tools, training for faculty members, and support to grow technical communities on campus.
- GitHub for your whole school, with everything you need to make it great
  - Free access to GitHub Enterprise Server and GitHub Enterprise Cloud for every department, including your technical departments
  - Teacher training to master Git and GitHub with our Campus Advisors program
  - Automated access to premium GitHub Education features, like the GitHub Student Developer Pack
  - Exclusive access to new features, and GitHub Education-specific swag

> What is Real World? :thinking: 
- Developing solution for world
- It is which impacts human race
- Which is agile
- Which develops ROI
- Helps creating lead for your career

> Different areas for contribution today

![image](https://user-images.githubusercontent.com/41017419/101821757-6866fd80-3b4e-11eb-83de-a0d06987fbec.png)

> Developer demand in growing even beyond tech

![image](https://user-images.githubusercontent.com/41017419/101822534-75382100-3b4f-11eb-836b-59f38c52652a.png)

> By 2031, demand for skilled workers will outstrip supply
- 85.2M global talent shortage
-  $8.5 Trillion unrealized annual revenue loss due to talent shortage
- 97M new jobs, 85M jobs will be replaced with new ones

> Innovating, Connecting and Solutioning with GitHub

![image](https://user-images.githubusercontent.com/41017419/101822911-0d360a80-3b50-11eb-8bf1-0c0cd470aab8.png)


> GitHub is the platform for rapid innovation
- Security & Compliance
- Code-to-cloud DevOps
- Team Collaboration
- Developer Talent
  - GitHub Developer Talent + attract and grow your teams
    - Largest and most diverse developer community filling gaps.
    - The new developer resume signalling passion and links to private contributions.
    - Data-driven learning, with signal from new trends and personalised content.

> GitHub Loves Students :heart: 
- 6 programs that serve teachers, students and institutes
- Served 2.5M students since 2012
- Poised to touch 10M students by 2024
- 650+ schools as campus program partners

- - -
If you have any questions, reach out to **[@DhirajGyani](https://twitter.com/DhirajGyani)**

> Do add your notes as well :smile:
